src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -115,6 +135,21 @@ protected boolean isElementDisplayed(WebElement element) {
   }
 
   /**
+   * Method to check if WebElement is displayed on the page
+   *
+   * @return true if element is displayed, otherwise return false
+   */
+
+  protected boolean isElementDisplayed(WebElement element, int timeout) {
+    try {
+      wait.forElementVisible(element, timeout);
+      return true;
+    } catch (TimeoutException e) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)